doc-content/enterprise-only/project-data/select-branches-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='select-branches-proc']
+
+= Selecting branches
+You can switch between branches to review previous project assets and to test new branch projects.
```

Comment:
I would rephrase this, instead of "to review previous project assets and to test new branch projects", I would use "to test them and review their assets".

Reply:
Changed as suggested.